src/server/routes/readiness/readiness.ts

```diff
@@ -63,8 +63,8 @@ const checkClusters = (clusters: Cluster[]): Promise<ClusterHealth[]> => {
 };
 
 const checkDruidCluster = (cluster: Cluster): Promise<ClusterHealth> => {
-  const { host } = cluster;
-  const loadStatusUrl = `http://${cluster.host}/druid/broker/v1/loadstatus`;
+  const { url } = cluster;
+  const loadStatusUrl = `${url}/druid/broker/v1/loadstatus`;
```

Comment:
It should be configurable finally (not necessarily in this PR).  But I didn't find a loadstatus endpoint for the Druid Router. Let's wait for the first use case when Turnilo is not connected to the broker.